lib/account.php

```diff
@@ -69,18 +72,19 @@ public function getImapConnection() {
 			$host = $this->account->getInboundHost();
 			$user = $this->account->getInboundUser();
 			$password = $this->account->getInboundPassword();
+			$password = $this->crypto->decrypt($password);
```

Comment:
during setup the password gets encrypted as well: https://github.com/owncloud/mail/blob/fix-244/lib/service/autoconfig.php#L384-L388